userdocs/contents/nodeconfig.rst

```diff
@@ -289,3 +289,18 @@ And to unset this configuration attribute:
 
    # wwctl node list -a n001 | grep Cluster
    n001                Cluster            --           --
+
+Important Considerations When Editing nodes.conf and warewulf.conf
+=================================================================
+
+When ``nodes.conf`` is edited, ``warewulfd`` does not know that the container profile has been changed. Therefore the new changes to ``nodes.conf`` are not taken into account until ``warewulfd`` is restarted.
+
+Once you restart ``warewulfd``, the ``nodes.conf`` file is then successfully reloaded.
+
+This also goes for ``warewulf.conf`` as well - any changes made also require ``warewulfd`` to be restarted.
```

Comment:
I suggest moving this to a different place in the docs (maybe https://warewulf.org/docs/main/contents/configuration.html). In fact, that might be a better place for _both_ of these notes, as both warewulf.conf and nodes.conf are mentioned there.

Reply:
Thank you kindly for taking a look @anderbubble and I agree, https://warewulf.org/docs/main/contents/configuration.html does mention both `warewulf.conf` and `nodes.conf`, so I will add those there. Yep, I will run `git commit --signoff` and amend this commit with the changes as a `.. note::`